src/schema/firefox-schemas-import.js

```diff
@@ -1,4 +1,5 @@
 import fs from 'fs';
+import { join as joinPath } from 'path';
```

Comment:
hmm, what's wrong with `import path from 'path';` and use `path.join(...)` instead?